### DISCUSSION TITLE
is is possible add golang 1.19rc1 to homebrew formula?

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#3420] [Casks]
I am golang devloper, the upcoming golang 1.19 will take us new Memory Mode, which may benefit our project. 
Shall we please add 1.19rc1 to homebrew formula
user can install it via 
```bash
brew install golang@1.19rc1
```
the golang 1.18 still be the default stable , so people won't hurt by it .

**Comment by carlocab** [comment]
We don't package release candidates because they don't meet our [criteria for acceptable formulae](https://docs.brew.sh/Acceptable-Formulae#stable-versions).

If you want the release candidate to be installable via a `brew install`, you can host that version in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/
